en/docs/get-started/about-this-release.md

```diff
@@ -91,16 +96,23 @@ The Micro Integrator dashboard can connect to multiple Micro Integrator server i
 
 It is now available to download from [here](https://github.com/wso2/micro-integrator/releases/).
 
-<! --- ### New features --->
+### New features
 
+- **[MI Dashboard Login Related Enhancements (SSO Capability)]({{base_path}}/install-and-setup/install/installing-the-product/running-the-mi-dashboard/#configuring-single-sign-on-with-openid-connect)**
+
+     Users can use OIDC based Single Sign-On to log in to the WSO2 Micro Integrator Monitoring Dashboard.
 
 ## **Streaming Integrator Tooling**
 
 The Streaming Integrator Tooling is a developer tool to develop Siddhi applications and simulate events for testing purposes.
 
 It is now available to download from [here](https://github.com/wso2/streaming-integrator-tooling/releases/).
 
-<! --- ### New features --->
+### New features
```

Comment:
Shall we add this under Streaming Integrator section..

Reply:
Done.